framework_tvb/tvb/adapters/analyzers/bct_centrality_adapters.py

```diff
@@ -45,32 +51,45 @@ class CentralityNodeBinary(BaseBCT):
     _ui_description = bct_description("betweenness_bin.m")
     _matlab_code = "C = betweenness_bin(A);"
 
+    def get_form_class(self):
+        return CentralityNodeBinaryForm
+
+    def launch(self, view_model):
+        connectivity = self.get_connectivity(view_model)
+        data = dict([('A', connectivity.weights)])
 
-    def launch(self, connectivity, **kwargs):
-        kwargs['A'] = connectivity.binarized_weights
-        result = self.execute_matlab(self._matlab_code, **kwargs)
+        result = self.execute_matlab(self._matlab_code, data=data)
         measure = self.build_connectivity_measure(result, 'C', connectivity,
                                                   "Node Betweenness Centrality Binary", "Nodes")
-        return [measure]
+        measure_index = self.load_entity_by_gid(measure.gid.hex)
```

Comment:
I think this just returns None. We should store the index at this step, not load it

Reply:
Also, calling .hex is not necessary here since inside the load_entity_by_gid we check the gid type and perform .hex if necessary

Reply:
made fix to store the ConnectivityMeasure